src/frontend/js/login.js

```diff
@@ -149,11 +223,31 @@ div:has(#submit_btn) {
     this.password = inputEl.value;
   }
 
+  controlloTastoInvio(e) {
```

Comment:
usiamo l'inglese per i nomi di classi, funzioni o variabili

Reply:
cambiato in checkEnterKey()